docs/user/lambda/index.rst

```diff
@@ -0,0 +1,23 @@
+Lambda Data Store
+=================
+
+The GeoMesa Lambda data store module is an implementation of the GeoTools ``DataStore`` that
+keeps data in two tiers, a transient tier and a persistent tier. The transient storage is backed by `Apache Kafka`_.
+Any other GeoMesa data store can be used for the persistent storage, although the current module only provides
+configuration for `Apache Accumulo`_. The Lambda data store is found in ``geomesa-lambda`` in the source distribution.
```

Comment:
I gave some loose examples in the 'overview' section. do you think I should also add something here? or expand the examples there?